coalahtml/tree.py

```diff
@@ -0,0 +1,65 @@
+from collections import OrderedDict
+from queue import Queue
+import os
+
+
+class Node:
+
+    def __init__(self, path):
+        self.child = {}
+        self.parent = None
+        self.path = path
```

Comment:
Yes it's always a name of file/directory. `name` would be appropriate I
guess?
ᐧ

On Thu, May 26, 2016 at 12:25 PM, Attila <notifications@github.com> wrote:

> In coalahtml/tree.py
> <https://github.com/coala-analyzer/coala-html/pull/46#discussion_r64698974>
> :
>
> > @@ -0,0 +1,65 @@
> > +from collections import OrderedDict
> > +from queue import Queue
> > +import os
> > +
> > +
> > +class Node:
> > +
> > +    def __init__(self, path):
> > +        self.child = {}
> > +        self.parent = None
> > +        self.path = path
>
> IIUC this path is always either filename or dirname, so maybe you can use
> a better name.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala-html/pull/46/files/8fc09d0dc5c2bd4bf2032aa0657b5a63af442363..2371e9ca763b5a8791588526ebd237d580b60c44#r64698974>
>